tests/tests_integration/test_api/test_documents.py

```diff
@@ -82,9 +82,12 @@ def test_list(
         text_file, _ = text_file_content_pair
         is_integration_test = filters.Prefix("externalId", _FILE_PREFIX)
 
-        documents = cognite_client.documents.list(limit=5, filter=is_integration_test)
+        documents = cognite_client.documents.list(
+            limit=5, filter=is_integration_test, sort=SortableDocumentProperty.mime_type
+        )
 
         assert len(documents) >= len(document_list)
+        assert sorted(doc.mime_type for doc in documents) == sorted(doc.mime_type for doc in document_list)
```

Comment:
shouldn't sort here if you want to check that server-side sort is working.
```suggestion
        assert [doc.mime_type for doc in documents] == sorted(doc.mime_type for doc in document_list)
```